src/test/java/HW21.java

```diff
@@ -0,0 +1,36 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+
+public class HW21 extends BaseTest {
+
+    @Test
+     public void renamePlaylist() throws InterruptedException {
+        login("zahinedu@gmail.com", "te$t$tudent");
+        editList();
+        newName();
+//        doesPlaylistExist();
+
+    }
+
+    public void editList() {
+      wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".playlist:nth-child(5)")));
+        WebElement editList = driver.findElement(By.cssSelector(".playlist:nth-child(5)"));
+       actions.doubleClick(editList).perform();
+
+    }
+   public  void newName() {
+        WebElement playlistInputField= wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("input[name='name']")))
+;
```

Comment:
**How can I beautify code in IntelliJ:**

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)